app/controllers/posts_controller.rb

```diff
@@ -4,11 +4,21 @@ def new
     @post = current_user.posts.new
   end
 
+  # def find
```

Comment:
hmmm is this for something in the future?

Reply:
Yes it is. Possibly for the edit functionallity